CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+
+## Unreleased
+
+### Breaking Changes
+- [\#355](https://github.com/cosmos/iavl/pull/355) `Get` in `iavlServer` no longer returns an error if the requested key does not exist. `GetResponse` now contains a `missing` boolean to indicate that a key does not exist, and the returned index will be that of the next occupied key.
```

Comment:
Why do we need that breaking change?

Reply:
I don't believe there is a supported way of returning structured error information in gRPC, so in order to return the index of the nearest key when accessing a missing key we have to change from an error response to a regular response. See original issue #354.